src/models/ViewConfigurationAdapter.ts

```diff
@@ -65,12 +68,17 @@ export default class ViewConfigurationAdapter {
       return false;
     }
 
+    if (object.seedDate && typeof object.seedDate === 'number') {
+      const seedNumber: number = object.seedDate as number;
+      object.seedDate = ViewConfigurationAdapter.sanitizedDate(seedNumber)
+    }
+
     if (
       // tslint:disable: no-string-literal
       object.dateType !== undefined &&
       [0, 1, 2, 3, 4, 5, 6, 7].includes(object.dateType) &&
       object.viewType !== undefined &&
-      [0, 1, 2, 3, 4, 5, 6, 7].includes(object.viewType)
+      [0, 1].includes(object.viewType)
```

Comment:
Feels like a lot of `return` statements in this method. Maybe break things up in a future refactor?

Reply:
good call!